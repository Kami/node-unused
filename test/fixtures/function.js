// unused function foo
function foo() {
};

// unused variable bar, but not baz
var bar = function baz() {
};

