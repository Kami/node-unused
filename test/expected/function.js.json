[
  {
    "name": "foo",
    "loc": {
      "line": 2,
      "column": 9
    },
    "param": false
  },
  {
    "name": "bar",
    "loc": {
      "line": 6,
      "column": 4
    },
    "param": false
  }
]